code/modules/research/designs/medical_designs.dm

```diff
@@ -563,13 +563,48 @@
 	surgery = /datum/surgery/advanced/viral_bonding
 	research_icon_state = "surgery_chest"
 
-/datum/design/surgery/reconstruction
-	name = "Reconstruction"
-	desc = "A surgical procedure that gradually repairs damage done to a body without the assistance of chemicals. Unlike classic medicine, it is effective on corpses."
-	id = "surgery_reconstruction"
-	surgery = /datum/surgery/advanced/reconstruction
+/datum/design/surgery/healing
+	name = "Tend Wounds"
+	desc = "An upgraded version of the original surgery."
```

Comment:
```suggestion
	desc = "An upgraded version of the original surgery."
	id = "surgery_healing_base" //holder because travis cries otherwise. Not used in techweb unlocks.
```